reproductions/SmokeTests.Core/ExitCode.cs

```diff
@@ -0,0 +1,8 @@
+namespace SmokeTests.Core
+{
+    public enum ExitCode : int
```

Comment:
Consider making these `const int ` fields on a `static` class, since it seems like we always need to cast them to `int` anyway
```csharp
public static class ExitCode
{
    public const int Success = 0;
    public const int UnknownError = -10;
}